PowerPointLabs/PowerPointLabs/PPMouseLegacy.cs

```diff
@@ -1,309 +0,0 @@
-﻿using System;
-using System.Diagnostics;
-using System.Runtime.InteropServices;
-using System.Windows.Forms;
-using System.Drawing;
-using PowerPointLabs;
-using PowerPoint = Microsoft.Office.Interop.PowerPoint;
-using System.Text;
-
-namespace PPExtraEventHelper
-{
-    internal class PPMouseLegacy
```

Comment:
It wasn't marked as deprecated, so I didn't know. What to do with those that depends on PPMouseLegacy?